README.md

```diff
@@ -43,6 +43,6 @@ I started with React because it's the one most obviously lacking a basic SSG, bu
 
 ## License
 
-Copyright © 2023 Matt Kane. Licenced under the MIT licence.
+Copyright © 2023 Matt Kane. Licensed under the MIT license.
```

Comment:
I'm British

Reply:
My apologies! My [Spellchecker](https://github.com/streetsidesoftware/vscode-spell-checker) highlighted and I auto-corrected without a second thought 🙈